agent/taskresource/volume/dockervolume.go

```diff
@@ -38,19 +34,36 @@ type VolumeResource struct {
 	lock sync.RWMutex
 }
 
+// DockerVolumeConfig represents docker volume configuration
+type DockerVolumeConfig struct {
+	Scope          string `json:"scope"`
```

Comment:
`go fmt` plz

Reply:
go fmt doesn't actually change this, interspersed comment line might be why? it doesn't align the field's name type json-tag columns.